tests/resources/prod_multi_config_test.yml

```diff
@@ -0,0 +1,49 @@
+label: 'test-production'
+
+default:
+  site: North
+  layout_name: test-layout
+  data_directory: ./
+  showers:
+    eslope: -2
+    zenith: 20 deg
+    phi: 0 deg
+  array:
+    model_version: Prod5
+    zenith: 20 deg
+    phi: 0 deg
+    default:
+      LST: 'D234'
+      MST: NectarCam-D
+    LST-01: '1'
+
+# These are the default settings for each primary for 20 deg zenith angle, La Palma.
```

Comment:
We haven't decided yet if we should put these CORSIKA parameters in the DB or keep a yaml file and how to organise it. This is the question we discussed a couple of weeks ago in the chat if you remember. The decision we made was to put the majority in the code actually as look up tables. Until I implement it I won't know if it's the best solution though.

Reply:
But there is a TODO here, two lines below this one. (And anyway it can't be forgotten, if you try simulating anything which is not La Palma 20 deg you will get an error.